examples/rosettacode/100_doors.nit

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env nit
+#
+# This file is part of NIT ( http://www.nitlanguage.org ).
+# This program is public domain
+
+
+
+class Door
+	#True : Open / False : Close
+	var state: Bool = false
+
+	fun is_open: Bool
```

Comment:
Even better, rename `state` to `is_open`. You will need a single attribute and one method.